### ISSUE TITLE
[Feature Request] Auto minimize user-defined apps

### ISSUE THREAD

**Issue by netsonicyxf** [sandwichdoge/MinimizeToTray#13]
Is it possible to allow user to add app's path to MinimizeToTray, 
so when user launch these apps, MinimizeToTray will minimize them automatically

**Comment by githerit** [comment]
I will support and add
the application really needs the option to save "minimized" applications
after each reboot, I have to put a tick in front of the applications I need to minimize

**Comment by TitusKirch** [comment]
That would be great. I have a program that I have in autostart, but by default it can't be minimized. Currently I always have to minimize it manually with MinimizeToTray after startup. If this would work automatically it would save me a few steps.

**Comment by FHMFightMe** [comment]
yeah, I was looking for a way to do this. I don't think it's possible though.

**Comment by jutuiz** [comment]
+1 
I was exactly looking for this.  
Example:  PDFxchange has a "Office to PDF" app that is running on default monitoring folders.  It's like always on, but has no minimize function build in.

**Comment by Diglador** [comment]
i instaled this app because i think it already can do that 
+1

Edit: 
if someone need this , python workaround idea example (without error handling):
import pyautogui
import os
#import ctypes
#import time

def send_to_mtt():
    pyautogui.keyDown('alt')
    pyautogui.press('F1')
    pyautogui.keyUp('alt') 

#os.startfile("C:\MinimizeToTray.exe")
#time.sleep(0.5)
#ctypes.windll.kernel32.SetConsoleTitleW("name_for_this_script")
#pyautogui.getWindowsWithTitle("name_for_this_script")[0].activate()
#send_to_mtt()
pyautogui.getWindowsWithTitle("AIMP")[0].activate()
send_to_mtt()
